content/python/concepts/strings/terms/isalnum/isalnum.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: '.isalnum()'
+Description: 'Takes a string and returns True if all the  characters are alphanumeric (A-Z, a-z, 0-9). Otherwise it returns False if the string contains any non-alphanumeric characters (@#$%-*)
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+The `.isalnum()` string method takes in a string and returns `True` if all the  characters are alphanumeric (A-Z, a-z, 0-9). Otherwise it returns `False` if the string contains any [non-alphanumeric](https://buildvirtual.net/a-guide-to-non-alphanumeric-characters/) characters (@#$%-*).
+
+## Syntax
+
+```py
+string.isalnum()
+```
+
+## Example
+
+Use `.isalnum()` to check if all the  characters are alphanumeric:
+
+```py
+my_string_1 = "string2023"
+
+print(my_string_1.isalnum())
+# Output: True
+
+my_string_2 = "@lpha"
+
+print(my_string_2.isalnum())
+# Output: False
+```
```

Comment:
For concepts that involve lots of examples we often comment the output. But, for most term entries we usually use a ```shell block.